bindata/manifests/daemon/daemonset.yaml

```diff
@@ -116,7 +116,7 @@ spec:
           command:
             - /bin/bash
             - -c
-            - mkdir -p /host/var/lib/sriov/ && cp /usr/bin/sriov-network-config-daemon /host/var/lib/sriov/sriov-network-config-daemon && chcon -t bin_t /host/var/lib/sriov/sriov-network-config-daemon | true # Allow systemd to run the file, use pipe true to not failed if the system doesn't have selinux or apparmor enabled
+            - mkdir -p /host/var/lib/sriov/ && cp /usr/bin/sriov-network-config-daemon /host/var/lib/sriov/sriov-network-config-daemon && chcon -t bin_t /host/var/lib/sriov/sriov-network-config-daemon | true  && cp /bindata/scripts/kargs.sh /host/var/lib/sriov/kargs.sh && chcon -t bin_t /host/var/lib/sriov/kargs.sh | true # Allow systemd to run the file, use pipe true to not failed if the system doesn't have selinux or apparmor enabled
```

Comment:
I think the original command has issue. Should it has `||` (or) instead of `|` (pipe)? The same question for the changes you added.

Reply:
yes, we need to fix it and we should we need to think how to simplify this line. thanks for pointing me on this!

Reply:
just tested again: we need only one `|` to not fail if `chcon` execution failed

Reply:
Yes, this will work because `|` will mask errors from the previous command (by default, if `set -o pipefail` is not set). The `||` looks cleaner to me, but I'm okay to keep the current implementation.